README.md

```diff
@@ -1,4 +1,4 @@
-# Web service with PHP/Symfony 
+# API base-project with PHP/Symfony 
```

Comment:
You can generate automatically helm configuration Table with https://github.com/norwoodj/helm-docs or others tools